geoh5py/ui_json/forms.py

```diff
@@ -453,23 +461,34 @@ def __init__(self, name, value=None, **kwargs):
             ],
         )
         self._is_value = BoolParameter("is_value")
-        self._property = UUIDParameter("property")
+        self._property = TypeRestrictedParameter(
+            "property", [self._data_type_string_to_type(data_type)]

```

Comment:
Right, the reason it's passed as a kwarg is because this pushes it through the .register defined on the super class that aggregates errors.  I just pushed an update that prevents a crash on the data_type lookup.   By setting NoneType as the default for the lookup, the TypeValidation will also catch a bad type and the two validations will be aggregated within register.  I added a test that confirms the suspected behaviour

Reply:
I see. Good for now, thanks for the change..